src/bin/ceresdb-server.rs

```diff
@@ -4,7 +4,7 @@
 
 // TODO(yingwen): ceresdb-server is a legacy name, maybe use a new name
```

Comment:
any thought?

Reply:
It was actually `cse-server is legacy name, maybe use a new name` so I will remove this TODO.